std_msgs/String algorithm
---
std_msgs/Bool result
